android/src/main/java/com/reactcommunity/rndatetimepicker/RNDismissableTimePickerDialog.java

```diff
@@ -31,28 +40,240 @@
  *   See: <a href="https://code.google.com/p/android/issues/detail?id=34833">Issue 34833</a>
  * </p>
  */
-public class RNDismissableTimePickerDialog extends TimePickerDialog {
+
+class CustomTimePickerDialog extends TimePickerDialog {
```

Comment:
Can you move this class to another java file?

Reply:
that's done